Flutter/datagrid/sorting.md

```diff
@@ -473,6 +473,91 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid shows customized the sort icon color](images/sorting/flutter-datagrid-customized-sorticon-color.png)
 
+## Set custom sorting icon
+
+`SfDataGrid` allows to change the sort icon through [SfDataGridTheme.SfDataGridThemeData](https://pub.dev/documentation/syncfusion_flutter_core/latest/theme/SfDataGridThemeData-class.html) property. The DataGrid should be wrapped inside the `SfDataGridTheme`. 
```

Comment:
`SfDataGrid` allows to change the sort icon through using `SfDataGridThemeData.sortIcon` property. The DataGrid should be wrapped inside the `SfDataGridTheme`.  The given icon will be shown with the size (16*16).